websites/H/HBO Max/presence.ts

```diff
@@ -47,14 +47,16 @@ const presence = new Presence({
 	},
 	coverUrls: Record<string, string> = {};
 
-/* eslint-disable camelcase */
-function getToken(): Promise<string> {
-	return new Promise(resolve => {
-		fetch("https://oauth.api.hbo.com/auth/tokens", {
+let isFetching = false;
+
+async function fetchToken(): Promise<string> {
+	const response = await (
+		await fetch("https://oauth.api.hbo.com/auth/tokens", {
```

Comment:
you should only have 1 await so you should keep the .then (it's used specifically for situations like these). In fact you should keep the two .then's and make the function not async so you only need to await it once when calling it